### PR TITLE
[GALLERY] - Gallery template base

### PR DESCRIPTION
**Pull request by bentranter**
This is a PR to add a starting point for the new Gallery template. It establishes the basic project layout, as well the base for a hero partial. Sections included so far:

- Main gallery page (gallery.amp.html)
- Inner gallery page (gallery-inner.amp.html)
- Hero partial (partials/hero.snip.html)

As this is just to establish a basic layout and project structure, there are no mocks or staging links at this point, but they will be included in subsequent PRs.

### CODE REVIEW

File: css/templates/gallery/page.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*! Bassplate | MIT License | http://github.com/basscss/bassplate */
+@import 'normalize.css';
+
+/* Basscss Modules */
+@import 'basscss-type-scale';
```
Comment (camelburrito) [inline css/templates/gallery/page.css]: just import core.css.

File: css/templates/gallery/page.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+body {
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  line-height: var(--line-height);
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
```
Comment (camelburrito) [inline css/templates/gallery/page.css]: why?

File: templates/gallery/partials/hero.snip.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#hero}}
+  <header>
```
Comment (camelburrito) [inline templates/gallery/partials/hero.snip.html]: why do we need this hero? are we going to add more stuff to this?

File: templates/gallery/gallery-inner.amp.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      "canonical-path": "templates/gallery-inner.amp",
+      "css-path": "templates/gallery/page.css",
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Poppins:400,500",
+        "https://fonts.googleapis.com/css?family=Oswald:400"
+      ]
+    },
+    "hero": {
+      "heading": "Living Tree",
+      "subheading": "<time datetime=\"2017-08-24\">Sept 24</time>-<time datetime=\"2017-08-29\">29</time> / Anna Moss",
```
Comment (camelburrito) [inline templates/gallery/gallery-inner.amp.json]: there is a byline component that can be used for this?

File: css/templates/gallery/page.css (hunk 2 above)
Comment (10xjs) [inline css/templates/gallery/page.css]: https://www.paulirish.com/2012/box-sizing-border-box-ftw/

File: templates/gallery/partials/hero.snip.html (hunk 3 above)
Comment (bentranter) [inline templates/gallery/partials/hero.snip.html]: Yeah, the next PR will add more content and styles to this hero.

File: css/templates/gallery/page.css (hunk 2 above)
Comment (camelburrito) [inline css/templates/gallery/page.css]: ok then lets change to use the new way in base-page.css

File: templates/gallery/gallery-inner.amp.json (hunk 4 above)
Comment (camelburrito) [inline templates/gallery/gallery-inner.amp.json]: i would prefer component reuse over rewrite. I just took a look at the component i think it is very much reusable -

File: templates/gallery/partials/hero.snip.html (hunk 3 above)
Comment (camelburrito) [inline templates/gallery/partials/hero.snip.html]: i think we should add it in the next PR then.

**Comment by bentranter** [comment]
Closing in favour of #576

**Pull request closed without merging**